src/qtt/qiskit/circuits.py

```diff
@@ -0,0 +1,45 @@
+from typing import List, Optional
+
+from qiskit.circuit.quantumcircuit import QuantumCircuit
+
+
+def integrate_circuit(
+    qc: QuantumCircuit,
+    target_qubits: List[int],
+    target_control_bits: Optional[List[int]] = None,
+    number_of_qubits: int = 5,
+    add_measurements: bool = True,
+) -> QuantumCircuit:
+    """ Integrate circuit at specified qubits in a larger qubit system
+
+    This can be used for example to integrate a single-qubit experiment in
```

Comment:
I doubt if this is always logical functionality. When the original circuit has 2-qubit gates and only one of them appears in target_qubits what happens then?

Reply:
That is not how the `target_qubits` argument works. The 2-qubit gate in a 3-qubit circuit has qubit indices 1, 2 in the original circuit and the target qubits are 1,3, 4 then the 2-qubit gate is applied on the resulting circuit on qubits 3, 4.